app/routes/circulars._archive._index.tsx

```diff
@@ -291,7 +340,152 @@ export default function () {
+                        value="month"
+                        label="Last Month"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      />
+                    </div>
+
+                    <div className="display-flex flex-column margin-1">
+                      <Radio
+                        id="radio-year"
+                        name="radio-date"
+                        value="year"
+                        label="Last Year"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      />
+                      <Radio
+                        id="radio-ytd"
+                        name="radio-date"
+                        value="ytd"
+                        label="Year to Date"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      />
+                      <Radio
+                        id="radio-custom"
+                        name="radio-date"
+                        value="custom"
+                        label="Custom Range..."
+                        onChange={(e) => {
+                          setShowDateRange(e.target.checked)
+                        }}
+                      />
+                    </div>
+                  </div>
+
+                  {showDateRange && (
+                    <DateRangePicker
+                      startDateHint="dd/mm/yyyy"
+                      startDateLabel="Start Date"
+                      className="margin-bottom-2"
+                      startDatePickerProps={{
+                        id: 'event-date-start',
+                        name: 'event-date-start',
+                        defaultValue: 'startDate',
+                        onChange: (value) => {
+                          setInputDateGte(value)
+                        },
+                      }}
+                      endDateHint="dd/mm/yyyy"
+                      endDateLabel="End Date"
+                      endDatePickerProps={{
+                        id: 'event-date-end',
+                        name: 'event-date-end',
+                        defaultValue: 'endDate',
+                        onChange: (value) => {
+                          setInputDateLte(value)
+                        },
+                      }}
+                    />
+                  )}
+                  <div className="display-flex flex-row">
+                    <Button
+                      type="button"
+                      className=""
+                      form="searchForm"
+                      onClick={() => {
+                        setDateRange()
+                      }}
+                    >
+                      <Icon.CalendarToday /> Submit
+                    </Button>
+                  </div>
+                </div>
```

Comment:
```suggestion
                </CardBody>
                <CardFooter>
                  <Button
                    type="button"
                    className=""
                    form="searchForm"
                    onClick={() => {
                      setDateRange()
                    }}
                  >
                    <Icon.CalendarToday /> Submit
                  </Button>
                </CardFooter>
```

Reply:
implemented this in 53a30ab, though I did notice that the submit button is slightly further right now 
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/57914086/13a4f5b6-bb5b-481a-b386-80942f09f6b3)